doc/architecture/software-update.rst

```diff
@@ -35,7 +63,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
 signing the file containing the references to the hashes.
-A secure channel is not required to distribute the updates.
+There is no need for a secure channel, for distributing the updates.
```

Comment:
The opposite is true at the moment: because the swupd client does no checking of the data that it receives, secure updates are only possible when using https and relying on securing the transport channel.

If the document is meant to describe the ultimate goal, then there should at least be a remark that explains the current state.